tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -133,49 +168,58 @@ bool RetryingFileSystem::FileExists(const string& fname) {
 
 Status RetryingFileSystem::Stat(const string& fname, FileStatistics* stat) {
   return CallWithRetries(
-      std::bind(&FileSystem::Stat, base_file_system_.get(), fname, stat));
+      std::bind(&FileSystem::Stat, base_file_system_.get(), fname, stat),
+      initial_delay_seconds_);
 }
 
 Status RetryingFileSystem::GetChildren(const string& dir,
                                        std::vector<string>* result) {
   return CallWithRetries(std::bind(&FileSystem::GetChildren,
-                                   base_file_system_.get(), dir, result));
+                                   base_file_system_.get(), dir, result),
+                         initial_delay_seconds_);
```

Comment:
Isn't the alignment right here? `std::bind(&FileSystem::GetChildren, base_file_system_.get(), dir, result)` is one parameter and `initial_delay_seconds_` is another.

Parameters to `std::bind` are aligned and parameters to `CallWithRetries` are algined